pylint/extensions/code_style.py

```diff
@@ -66,6 +66,7 @@ class CodeStyleChecker(BaseChecker):
             "max-line-length-suggestions",
             {
                 "type": "int",
+                "default": 0,
```

Comment:
I noticed the default value that get added, I think it's a very nice perks of the migration.

Reply:
Yeah my `argparse` implementation errors out when stuff like `default` or `metavar` is missing.